### PR TITLE
Update SDK and fix code scanning alerts

### PR DESCRIPTION
**Pull request by elopezanaya**
### Description
_Include a description of the problem to be solved_

https://github.com/microsoft/omnichannel-chat-widget/security/code-scanning/3 [Added to changelog]
https://github.com/microsoft/omnichannel-chat-widget/security/code-scanning/21 [Not a product change]

## Solution Proposed
_Detail what is the solution proposed, include links to design document if required or any other document required to support the solution_

### Acceptance criteria
_Define what are the conditions to consider the PR has achieved the intended goal_

## Test cases and evidence
_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_

### No side effect email validation

image

image

### Session  without issues

image

### Sanity Tests
-   [ ] You have tested all changes in Popout mode
-   [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
-   [ ] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y
- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

__*Please provide justification if any of the validations has been skipped.*__

### CODE REVIEW

File: chat-widget/package.json [hunk 1]
```diff
@@ -87,7 +87,7 @@
     "@azure/core-tracing": "^1.2.0",
     "@microsoft/applicationinsights-web": "^3.3.6",
     "@microsoft/omnichannel-chat-components": "1.1.12",
-    "@microsoft/omnichannel-chat-sdk": "^1.11.2",
+    "@microsoft/omnichannel-chat-sdk": "^1.11.3",
```
Comment (Sonam-Ligade) [inline chat-widget/package.json:90]: updated lock file?
